apps/backend/fixtures/lambda/reconcile.json

```diff
@@ -3,12 +3,9 @@
     {
       "Sns": {
         "Message": {
-          "period": {
-            "from": "2023-01-01",
-            "to": "2023-01-02"
-          },
-          "program": "SBC", 
-          "generateReport": true, 
+          "reconciliationMaxDate": "2023-08-14",
```

Comment:
I'm confused why this PR is touching the scope of removing the start date from reconciliation (switching from period to only a max date).

Reply:
We've had some discussions where we felt a start date wasn't super necessary anymore - IMO if we're making an attempt to reconcile, we should be reconciling the data we have regardless of when they started. The date range was really for supporting batch reconcile, which is something I think we're focusing less on, opting to focus on the real world scenario of saying "reconcile up until this date" (typically "yesterday").

Reply:
We 100% have discussed changing this but it's not in the PR description or part of the ticket, so I'm not super sure on the state of things - did we already remove other code related to this and this PR is some missed cleanup? Or is this PR the functional change and we'll need corresponding documentation/signoff of that?

Reply:
The requirements for running reconciliation in production only require one date, but for internal and testing purposes we still need the flexibility to reset data and reconcile in batches. I should have made this change with the initial work on splitting out the test/batch logic from the production flow but I missed the opportunity then. 

TBH we might want to remove this input altogether as reconciliation should really only be called from the parser directly in prod, and/or from the batch-reconcile handler if batching in dev/test. 

The only use case I see for keeping these inputs is to individually trigger the lambda for debugging purposes.

Reply:
Logged https://bcdevex.atlassian.net/browse/CCFPCM-618 to clean this up in future.